pkg/controllers/allocation/bind.go

```diff
@@ -61,12 +61,13 @@ func (b *Binder) Bind(ctx context.Context, node *v1.Node, pods []*v1.Pod) error
 	}
 
 	// 4. Bind pods
-	logging.FromContext(ctx).Infof("Binding %d pod(s) to node %s", len(pods), node.Name)
 	errs := make([]error, len(pods))
 	workqueue.ParallelizeUntil(ctx, len(pods), len(pods), func(index int) {
 		errs[index] = b.bind(ctx, node, pods[index])
 	})
-	return multierr.Combine(errs...)
+	err := multierr.Combine(errs...)
+	logging.FromContext(ctx).Infof("Bound %d/%d pod(s) to node %s", len(pods) - len(multierr.Errors(err)),  len(pods), node.Name)
```

Comment:
I don't know if we need fractions here. I think it'd be sufficient to just say how many did bind.